web/views.py

```
@@ -188,6 +186,10 @@
 # Initialize Stripe
 stripe.api_key = settings.STRIPE_SECRET_KEY
 
+# Test media view for testing purposes
+def test(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `test`

(ANN201)